tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Common/MatterProvision.cs

```diff
@@ -1510,7 +1528,10 @@ private static string TeamMembersPermissionInformation(MatterDetails matterDetai
                 }
                 propertyList.Add(matterSettings.StampedPropertyMatterName, WebUtility.HtmlEncode(matter.Name));
                 propertyList.Add(matterSettings.StampedPropertyMatterID, WebUtility.HtmlEncode(matter.Id));
-                propertyList.Add(matterSettings.StampedPropertyClientName, WebUtility.HtmlEncode(client.Name));
+                if(!propertyList.ContainsKey(matterSettings.StampedPropertyClientName))

```

Comment:
For PC, the TeamName and ClientName are same. So before we add client name to the stamped properties, we are checking whether that has been already added or not. If not, we will add client name to stamped properties